doc/Programming-guide/Using-Kaa-endpoint-SDKs/Java/index.md

```diff
@@ -3,16 +3,201 @@ layout: page
+    <version>${build-version}</version>
+    <scope>system</scope>
+    <systemPath>${project.basedir}/lib/kaa-java-ep-sdk-LVcpSA4q5BiErcm10Zg86TSGL9s.jar</systemPath>
+</dependency>
+```  
+</div>
+<div id="gradle-sdk" class="tab-pane fade" markdown="1">
+```groovy
+dependencies {
+   compile  files('./lib/kaa-java-ep-sdk-LVcpSA4q5BiErcm10Zg86TSGL9s.jar')
+}
+```
+</div>
+</div>
+
+Also you need add following dependencies to enable logging:
+
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#Maven">Maven</a></li>
+  <li><a data-toggle="tab" href="#Gradle">Gradle</a></li>
+</ul>
+
+<div class="tab-content">
+
+<div id="Maven" class="tab-pane fade in active" markdown="1">
+```xml
+<dependency>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-api</artifactId>
+    <version>1.7.7</version>
+</dependency>
+
+<dependency>
+    <groupId>ch.qos.logback</groupId>
+    <artifactId>logback-core</artifactId>
+    <version>1.1.2</version>
+</dependency>
+
+<dependency>
+    <groupId>ch.qos.logback</groupId>
+    <artifactId>logback-classic</artifactId>
+    <version>1.1.2</version>
+</dependency>
+```  
+</div>
+<div id="Gradle" class="tab-pane fade" markdown="1">
+
+```groovy
+compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.7'
+
+compile group: 'ch.qos.logback', name: 'logback-core', version: '1.1.2'
+
+compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.1.2'
+```
+</div>
+</div>
+
+After configuring all this stuff you can start to write your client.
+
+## Base API overview
+
+First, you have to create <code>KaaClient</code>. SDK provide you class factory  <code>Kaa</code> that responsible for creating new instance of client.
+
+So, hear example code how to do this :
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#java">Java</a></li>
+  <li><a data-toggle="tab" href="#android">Android</a></li>
+</ul>
+
+
+<div class="tab-content">
+<div id="java" class="tab-pane fade in active" markdown="1">
+
+```java
+KaaClient client = Kaa.newClient(new DesktopKaaPlatformContext(), new SimpleKaaClientStateListener());
+```
+</div>
+
+<div id="android" class="tab-pane fade" markdown="1">
+
+```java
+KaaClient client = Kaa.newClient(new AndroidKaaPlatformContext(), new SimpleKaaClientStateListener());
+```
+</div>
+</div>
+
+Static method <code>newClient()</code> received two mandatory arguments &ndash; implementation of platform context and state listener.
+
+First argument is <code>DesktopKaaPlatformContext</code> for desktop or <code>AndroidKaaPlatformContext</code> for mobile applications.
+
+Second argument is out of the box implementation of <code>KaaClientStateListener</code> &ndash; <code>SimpleKaaClientStateListener</code> that just logged current state of Kaa client. 
+After getting the client, call method <code>start()</code> to begin communication with server. Now, you can use features provided by Kaa platform like data collection, notifications and etc.
```

Comment:
When the new instance of the client is created the method <code>start()</code> should be invoked in order to start client operation and communication with server. Starting from this point you can use features API provided by Kaa platform such as data collection, notifications and etc.